Exiled.API/Features/Map.cs

```diff
@@ -525,6 +525,22 @@ public static GameObject PlaceTantrum(Vector3 position)
         /// <param name="transmitterId">Sets the transmitterId.</param>
         public static void PlayIntercomSound(bool start, int transmitterId = 0) => Intercom.host.RpcPlaySound(start, transmitterId);
 
+        /// <summary>
+        /// Places a blood decal.
+        /// </summary>
+        /// <param name="position">The position of the blood decal.</param>
+        /// <param name="type">The <see cref="BloodType"/> to place.</param>
+        /// <param name="multiplier">A value which determines the spread of the blood decal.</param>
+        public static void PlaceBlood(Vector3 position, BloodType type, float multiplier = 1f) => PlayerManager.hostHub.characterClassManager.RpcPlaceBlood(position, (int)type, multiplier);
+
+        /// <summary>
+        /// Gets all the near cameras.
+        /// </summary>
+        /// <param name="position">The position from which starting to search cameras.</param>
+        /// <param name="toleration">The maximum toleration to define the radius from which get the cameras.</param>
+        /// <returns>A <see cref="IEnumerable{T}"/> of <see cref="Camera"/> which contains all the found cameras.</returns>
+        public static IEnumerable<Camera> GetNearCameras(Vector3 position, float toleration = 15f) => Cameras.Where(cam => Vector3.Distance(position, cam.Position) <= toleration);
```

Comment:
This is probably a faster method.
```cs
Cameras.Where(cam => (position - cam.Position).sqrMagnitude <= toleration * toleration);
```

Reply:
It's just a micro-optimization which is not needed, and sometimes it's not even different.

Reply:
It is a micro-optimization but it's also just about **always** faster because it's easier to multiple 2 numbers together than it is to calculate a square root